### PR TITLE
more fixes suggested by cython-lint

### PR DESCRIPTION
**Pull request by fchapoton**
This is just fixing a few details in origami, as suggested by cython-lint. Typically unused variables, code formatting, etc

Found using
`
cython-lint --ignore=E501,E261,E222,E265,E266,E127,E128,E231,E741 surface_dynamics
`

### CODE REVIEW

File: surface_dynamics/flat_surfaces/origamis/origami_dense.pyx [hunk 1]
```diff
@@ -2069,7 +2074,6 @@ cdef class Origami_dense_pyx:
         if self._n != other._n:
             return False
         cdef Origami_dense_pyx ss, oo
-        cdef ms, mo
```
Comment (saraedum) [inline surface_dynamics/flat_surfaces/origamis/origami_dense.pyx:2072]: @videlec was there a reason for this `cdef`?

**Comment by fchapoton** [comment]
Should I add back this line `cdef ms, mo` ? Its removal was suggested by `cython-lint`.

It looks a bit strange to me, as it is not giving a precise C type.

File: surface_dynamics/flat_surfaces/origamis/origami_dense.pyx (hunk 1 above)
Comment (videlec) [reply surface_dynamics/flat_surfaces/origamis/origami_dense.pyx:2072]: I think that I always prefer declaring my variables. Nothing more.

**Comment by saraedum** [comment]
> Should I add back this line `cdef ms, mo` ? Its removal was suggested by `cython-lint`.
> 
> It looks a bit strange to me, as it is not giving a precise C type.

I don't think we need these declarations.

**Review by saraedum — APPROVED** [review]

**Pull request merged**